js/src/redux/providers/status.js

```diff
@@ -102,7 +102,7 @@ export default class Status {
       }, timeout);
     };
 
-    fetch('/', { method: 'HEAD' })
+    fetch('/api/ping', { method: 'HEAD' })
```

Comment:
Anything under `//api/*` is not available on :8180, needs to be /. You cannot connect to :8080 either since to get the port you first need the API up via dappsUrl.